keadm/cmd/keadm/app/cmd/cloud/upgrade.go

```diff
@@ -0,0 +1,69 @@
+package cloud
+
+import (
+	"github.com/spf13/cobra"
+
+	types "github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
+	"github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/helm"
+)
+
+func NewCloudUpgrade() *cobra.Command {
+	opts := newCloudUpgradeOptions()
+
+	cmd := &cobra.Command{
+		Use:   "cloud",
+		Short: "Upgrade the cloud components",
+		Long:  "Upgrade the cloud components to the desired version, it uses helm to upgrade the installed release of cloudcore.",
```

Comment:
> it uses helm to upgrade the installed release of cloudcore.

Other components should also use helm to upgrade?

Reply:
Yes, cloud components cloudcore, controllermanager, admission and iptablemanager are all in the cloudcore chart.

Reply:
```suggestion
		Long:  "Upgrade the cloud components to the desired version, it uses helm to upgrade the installed release of cloudcore chart, which includes all the cloud components.",
```
Okey, I think we can add some description for the cloudcore chart, or user will be confused, think we only upgrade the cloudcore.

Reply:
Done, thanks.